hugegraph-test/src/main/java/com/baidu/hugegraph/unit/UnitTestSuite.java

```diff
@@ -119,6 +120,7 @@
 
     /* rocksdb */
     RocksDBSessionsTest.class,
+    RocksDBSessionTest.class,
```

Comment:
also add UnitUtilTest

Reply:
removed it and to import from common module